helm-build-push/README.md

```diff
@@ -32,6 +32,7 @@ jobs:
 | gpg-secret-name         | Gpg secret key name from gpg secret key. Needed to use gpg sign.                | Optional |
 | gpg-secret-key          | Base64 encoded gpg secret key for chart sign. Needed if gpg-secret-name is set. | Optional |
 | gpg-secret-key-password | The password for the gpg secret key. Needed if gpg-secret-name is set.          | Optional |
+| enable-chart-test       | Enable testing of Helm charts.                                                  | Optional |
```

Comment:
| enable-chart-test       | Enable testing of Helm charts.  Default is true                                                | Optional |